### ISSUE TITLE
val loss is extreamly bigger than training loss, why? 

### ISSUE THREAD

**Issue by claylau** [bardiadoosti/HOPE#33]
I have done training some epoch (less 1000) on the whole HopeNet and Resnet10 part separately，i find training loss is going to converge at begining  but val loss seem to falling more slowly and is extreamly larger than training loss **even to 100 times**.

trainging settings keep same (with bach size=64, learning rate=1e-3), i am not sure what happened ? Thanks for your help.
